README.md

```diff
@@ -40,6 +40,7 @@ make
   Consul API queries to gather all information about each service. Health check
   information are available via `consul_health_service_status` as well, but
   only for services which have a health check configured. Defaults to true.
+* __`consul.health-status-code`:__  Returns a health status code for passing (0), warning (1) or critical (2) via `consul_health_service_status` and `consul_catalog_service_node_healthy`. Default health status code is passing (1) and critical/warning (0).
```

Comment:
Returning 0 for passing and 1 for non-healthy for a metric called `..._healthy` is unidiomatic and confusing.

Furthermore, the change is not longer consistent with the metrics HELP texts.